src/imitation/util/util.py

```diff
@@ -64,6 +64,7 @@ def make_vec_env(env_id: str,
       log_dir: If specified, saves Monitor output to this directory.
   """
   def make_env(i):
+    import imitation.examples.env_suite  # noqa: F401
```

Comment:
Best to have imports be top-level where possible, see http://google.github.io/styleguide/pyguide.html#313-imports-formatting